core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -836,6 +836,14 @@ public Integer getSchedulerPollIntervalSecond() {
         return Integer.parseInt(getOptional("kylin.job.scheduler.poll-interval-second", "30"));
     }
 
+    public boolean isSchedulerSafeMode() {
+        return Boolean.parseBoolean(getOptional("kylin.job.scheduler.safemode", "false"));
+    }
+
+    public String getSchedulerSafeModeExecutableUUID() {
```

Comment:
I think It's bettter to have a whitelist of project other than uuid. The job uuid may be changed.

Reply:
change whitelist to project name